include/exercise_functions.php

```diff
@@ -0,0 +1,504 @@
+<?php
+
+function addExercise(){
+
+  $exerciseTypeId = getExerciseTypeId();
+
+  $exerciseName = $_REQUEST["exerciseName"];
+  $session = $_REQUEST["session"];
+  $exerciseTarget = $_REQUEST["target"];
+  $userId = $_SESSION['userId'];
+
+  debugOutput($_REQUEST);
+
+  db_Query("
+    INSERT INTO exercise(exerciseName, typeId, session, target, userId)
+    VALUES(:exerciseName, :typeId, :session, :target, $userId)
+  ",
+  
+  [
+    'exerciseName' => $exerciseName,
+    'typeId' => $exerciseTypeId,
+    'session' => $session,
+    'target' => $exerciseTarget
+  ]
+
+  
+  );
+}
+
+function getExerciseTypes(){
+
+  $exerciseTypeData = db_Query("
+    SELECT *
+    FROM exercise_type
+  ")->fetchAll();
+
+  return $exerciseTypeData;
+
+}
+
+function getExerciseTypeId(){
+
+  $exerciseTypeName = $_REQUEST["exerciseType"];
+
+  $exerciseTypeId = db_Query("
+    SELECT typeId
+    FROM exercise_type
+    WHERE typeName = '$exerciseTypeName'
+  ")->fetch();
+
+  return $exerciseTypeId['typeId'];
+
+}
+
+function getExerciseTypeName($individualExercise){
+
+  $typeId = $individualExercise['typeId'];
+
+  $exerciseTypeName = db_Query("
+    SELECT typeName
+    FROM exercise_type
+    WHERE typeId = $typeId
+  ")->fetch();
+
+  return $exerciseTypeName['typeName'];
+
+}
+
+function displayExerciseTypeOptions(){
+
+  $exerciseTypeData = getExerciseTypes();
+  $count = 1;
+    
+
+  foreach($exerciseTypeData as $individualExerciseType){
+
+    $typeId = $individualExerciseType['typeId'];
+    
+    echo"
+      <option value=".$individualExerciseType['typeName'].">".$individualExerciseType['typeName']."</option>
+    ";
+
+  }
+
+}
+
+
+function getExercises(){
+
+  $userId = $_SESSION['userId'];
```

Comment:
I would probably pass in `userId` as a parameter of this function instead of relying on the `$_SESSION` being set